modules/apps/document-library/document-library-taglib/src/main/java/com/liferay/document/library/taglib/servlet/taglib/RepositoryBrowserTag.java

```diff
@@ -120,24 +120,24 @@ _folderModelResourcePermission, _getFolderId(),
 				portletRequest, _getRepositoryId(), getFolderId()));
 	}
 
-	private Set<String> _getActions() {
+	private Set<String> _getActionsSet() {
 		if (Validator.isBlank(getActions())) {
 			return _allActions;
 		}
 
 		String actions = getActions();
 
-		Set<String> set = SetUtil.fromArray(actions.split(",\\s*"));
+		Set<String> actionsSet = SetUtil.fromArray(actions.split("\\s*,\\s*"));
```

Comment:
This regular expression doesn't take into account actions at the start or the end of the actions string if string starts or ends with spaces.